Dangerfile.ts

```diff
@@ -1,7 +1,6 @@
 // import custom DangerJS rules
 // see http://danger.systems/js
 // see https://github.com/teamdigitale/danger-plugin-digitalcitizenship/
-// tslint:disable-next-line:prettier
```

Comment:
this line can be removed since eslint is set to ignore the Dangerfile in .eslintrc.js